source/server/listener_manager_impl.cc

```diff
@@ -668,12 +668,12 @@ void ListenerManagerImpl::drainListener(ListenerImplPtr&& listener) {
   draining_it->listener_->localDrainManager().startDrainSequence([this, draining_it]() -> void {
     draining_it->listener_->debugLog("removing listener");
     for (const auto& worker : workers_) {
-      // Once the drain time has completed via the drain manager's timer, we tell the workers to
-      // remove the listener.
+      // Once the drain time has completed via the drain manager's timer, we tell the workers
+      // to remove the listener.
       worker->removeListener(*draining_it->listener_, [this, draining_it]() -> void {
-        // The remove listener completion is called on the worker thread. We post back to the main
-        // thread to avoid locking. This makes sure that we don't destroy the listener while filters
-        // might still be using its context (stats, etc.).
+        // The remove listener completion is called on the worker thread. We post back to the
```

Comment:
This is not intended, but when I ran `./ci/run_envoy_docker.sh './ci/do_ci.sh fix_format'` - it automatically formatted this.